src/gp_doctest.cpp

```diff
@@ -7,7 +7,7 @@ using namespace GPOperations;
 
 // GPCSP stands for generalized PCSP-- see text.
 // Let the "venus" node be the common ancestor of mars and saturn.
-enum HelloGPCSP { jupiter, mars, saturn, venus, root };
+enum HelloGPCSP { jupiter, mars, saturn, root, venus };
```

Comment:
This order surprises me. Shouldn't the root have the largest ID?

Reply:
I agree, I debated about this with myself few times as it seemed unnatural. If you also think it's unnatural, then I will change it. But this, together with bringing back old tests will take a bit more time.

Reply:
Done.